web3-wrapper/src/web3_wrapper.ts

```diff
@@ -715,19 +714,27 @@ export class Web3Wrapper {
      * @return The contents nested under the result key of the response body
      */
     public async sendRawPayloadAsync<A>(payload: Partial<JSONRPCRequestPayload>): Promise<A> {
-        const sendAsync = this._provider.sendAsync.bind(this._provider);
+        if (!payload.method) {
+            throw new Error(`Must supply method in JSONRPCRequestPayload, tried: [${payload}]`);
+        }
+        const sendAsync = promisify(this._provider.sendAsync.bind(this._provider));
+        // tslint:disable:no-object-literal-type-assertion
         const payloadWithDefaults = {
             id: this._jsonRpcRequestId++,
             params: [],
             jsonrpc: '2.0',
             ...payload,
-        };
-        const response = await promisify<JSONRPCResponsePayload>(sendAsync)(payloadWithDefaults);
-        if (response.error) {
-            throw new Error(response.error.message);
```

Comment:
Don't we still want to throw on this?

Reply:
I kinda tested this, it looks like `await sendAsync` will throw, like a normal `await {PromiseReturningFunction}` when the promise rejects. 

Looking at this code again though I think the error handling doesn't make sense, will update. 🙏